### PR TITLE
avoid globals

### PR DESCRIPTION
**Pull request by freben**

### CODE REVIEW

**Review by Rugvip — APPROVED** [review]
Nice! 😁

File: plugins/github-actions/src/components/WorkflowRunDetails/WorkflowRunDetails.tsx [hunk 1]
```diff
@@ -116,8 +116,6 @@ const JobListItem = ({
     <Accordion TransitionProps={{ unmountOnExit: true }} className={className}>
       <AccordionSummary
         expandIcon={<ExpandMoreIcon />}
-        aria-controls={`panel-${name}-content`}
```
Comment (agentbellnorm) [inline plugins/github-actions/src/components/WorkflowRunDetails/WorkflowRunDetails.tsx:119]: Why are we removing these?

**Review by agentbellnorm — APPROVED** [review]
Cool!

Approving to unblock but i had a small question.

File: plugins/github-actions/src/components/WorkflowRunDetails/WorkflowRunDetails.tsx (hunk 1 above)
Comment (freben) [reply plugins/github-actions/src/components/WorkflowRunDetails/WorkflowRunDetails.tsx:119]: Ah yeah good catch! See [this discussion](https://github.com/backstage/backstage/pull/14778#discussion_r1029333583), which I totally should have linked to in the PR to explain myself.

**Pull request merged**